example/service.js

```diff
@@ -32,4 +32,50 @@ module.exports = async function() {
     TrackPlayer.destroy()
   });
 
+  let playingBeforeDuck;
+  let volumeBeforeDuck;
+  TrackPlayer.addEventListener(
+    'remote-duck',
+    async ({ paused, permanent, ducking }) => {
+      // When the event is triggered with permanent set to true,
+      // you should stop the playback:
+      if (permanent) {
+        TrackPlayer.stop();
+        return;
+      }
+
+      // When the event is triggered with ducking set to true,
+      // you should either lower the volume or pause the app.
+      // If it’s a music app, you probably want to lower the volume,
+      // but if it’s a podcast app, you probably want to pause it for a moment.
+      if (ducking) {
+        volumeBeforeDuck = await TrackPlayer.getVolume();
```

Comment:
Manually ducking has been removed in favor of letting the OS do the ducking automatically, the `ducking` parameter has also been removed with that.